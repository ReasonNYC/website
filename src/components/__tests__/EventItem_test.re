open Jest;

open ExpectJs;

let _ =
  describe
    "src/app.re"
    (
      fun _ =>
        test
          "renders"
          (
            fun _ => {
              let tree =
                EventItem.createElement
                  title::"Foo Title"
                  time::11234412.0
                  link::"https://google.com"
                  id::"1234"
                  description::"Hello World!"
                  children::[]
                  () |> ReactShallowRenderer.renderWithRenderer;
              expect tree |> toMatchSnapshot
            }
          )
    );
