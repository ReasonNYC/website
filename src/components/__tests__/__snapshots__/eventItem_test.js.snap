// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/app.re renders 1`] = `
<li
  className="EventItem"
>
  <h3
    className="EventItem-title"
  >
    <a
      href="https://google.com"
      rel="noopener"
      target="_blank"
    >
      Foo Title
    </a>
  </h3>
  <time
    className="EventItem-date"
  >
    1969-12-31 22:07:14
  </time>
  <div
    className="EventItem-description"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Hello World!",
      }
    }
  />
</li>
`;
